src/Compiler/perf/Microsoft.AspNetCore.Razor.Microbenchmarks.Generator/Microsoft.AspNetCore.Razor.Microbenchmarks.Generator.csproj

```diff
@@ -31,7 +31,6 @@
   <ItemGroup Condition="'$(Configuration)' == 'Release_Nuget'">
     <PackageReference Include="Microsoft.NET.Sdk.Razor.SourceGenerators.Transport" GeneratePathProperty="true" />
     <Reference Include="$(PkgMicrosoft_NET_Sdk_Razor_SourceGenerators_Transport)\source-generators\*.dll" />
-
   </ItemGroup>
-
-</Project>
+  
```

Comment:
Spaces?

Reply:
@333fred What do you mean by spaces? The gap between the last item group and the closing project element follows the style of the rest of the repo.

Reply:
I mean that when I look at this in the commit diff, I see spaces:
![image](https://user-images.githubusercontent.com/2371880/205747426-14451cd5-392e-4396-b236-5e26e5d05156.png)